### PR TITLE
Verifier is now correctly publishing

### PR DESCRIPTION
**Pull request by Clintonio**
Why: Cordapps broken on master due to the verifier not publishing correctly.

### CODE REVIEW

**Comment by Clintonio** [comment]
I'll have to add this to Bintray too.

File: node/src/main/kotlin/net/corda/node/driver/Driver.kt [hunk 1]
```diff
@@ -64,6 +64,7 @@ interface DriverDSLExposedInterface {
      * @param providedName Optional name of the node, which will be its legal name in [Party]. Defaults to something
      *   random. Note that this must be unique as the driver uses it as a primary key!
      * @param advertisedServices The set of services to be advertised by the node. Defaults to empty set.
+     * @param verifierType The type of contract verifier to use. See: [VerifierType]
```
Comment (exFalso) [inline node/src/main/kotlin/net/corda/node/driver/Driver.kt]: This is nitpicking, but I think we should be consistent, this is for *transaction* verification.

Comment (Clintonio) [inline node/src/main/kotlin/net/corda/node/driver/Driver.kt]: That's fair, no point in merging an inconsistent docs fix 👍

**Comment by Clintonio** [comment]
Updated @exFalso

**Pull request merged**